subprojects/core/src/main/java/org/gradle/api/tasks/bundling/AbstractArchiveTask.java

```diff
@@ -124,7 +124,7 @@ public Property<String> getArchiveFileName() {
     public Provider<RegularFile> getArchiveFile() {
         // TODO: Turn this into an `@implSpec` annotation on the comment above:
         // https://github.com/gradle/gradle/issues/7486
-        /**
+        /*
```

Comment:
🤔 given how markup-heavy this paragraph is, I wonder if removing the javadoc from it is a good idea.

Reply:
according to errorprone this is not a place for javadoc.

So even if it was meant to be one, it would never show up anywhere, or would it?

Reply:
>I think it is as good as with it

I don't have particularly strong feelings about this one, let's proceed with what we have now.